lib/facility.rb

```diff
@@ -1,14 +1,64 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
-
-  def initialize(name, address, phone)
+  attr_reader :name, :address, :phone,
```

Comment:
So for attr_readers, we want them either all on one line OR, one line for each - listed vertically. This format is too hard to read. I like the vertical format where each attr_reader would be on it's own line.